test/api-e2e/api_test.go

```diff
@@ -57,10 +61,12 @@ func (s *APISuite) TestISBSVC() {
 	assert.Contains(s.T(), listISBSVCBody, testISBSVCName)
 
 	getISBSVCBody := HTTPExpect(s.T(), "https://localhost:8443").GET(fmt.Sprintf("/api/v1/namespaces/%s/isb-services/%s", Namespace, testISBSVCName)).
+		WithMaxRetries(5).
```

Comment:
This retry and timeout will not work. They are for single http request, what we need retry if the result is not expected.